DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/Resources.resx

```diff
@@ -0,0 +1,281 @@
+  </data>
+  <data name="ActivityNameAnalyzerTitle" xml:space="preserve">
+    <value>Activity Function call references unknown Activity Function</value>
+  </data>
+  <data name="ActivityReturnTypeAnalyzerDescription" xml:space="preserve">
+    <value>Activity Function call return type doesn't match function definition return type</value>
+  </data>
+  <data name="ActivityReturnTypeAnalyzerMessageFormat" xml:space="preserve">
+    <value>Activity function named '{0}' returns '{1}' but '{2}' is expected</value>
+  </data>
+  <data name="ActivityReturnTypeAnalyzerTitle" xml:space="preserve">
+    <value>Activity Function call return type doesn't match function definition return type</value>
+  </data>
+  <data name="DateTimeAnalyzerTitle" xml:space="preserve">
+    <value>DateTime calls must be deterministic inside an orchestrator</value>
+  </data>
+  <data name="DeterministicAnalyzerDescription" xml:space="preserve">
+    <value>Orchestrator must be deterministic. For more information on orchestrator code constraints, see:
+https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-checkpointing-and-replay#orchestrator-code-constraints</value>
+  </data>
+  <data name="DeterministicAnalyzerMessageFormat" xml:space="preserve">
+    <value>'{0}' violates the orchestrator deterministic code constraint</value>
+  </data>
+  <data name="DispatchClassNameAnalyzerDescription" xml:space="preserve">
+    <value>DispatchAsync must be called with the Entity Class it's used in</value>
+  </data>
+  <data name="DispatchClassNameAnalyzerMessageFormat" xml:space="preserve">
+    <value>DispatchAsync was used with '{0}' but must be called with the Entity Class '{1}' it's used in</value>
+  </data>
+  <data name="DispatchClassNameAnalyzerTitle" xml:space="preserve">
+    <value>DispatchAsync must be called with the Entity Class it's used in</value>
+  </data>
+  <data name="DurableClientAnalyzerDescription" xml:space="preserve">
+    <value>DurableClient attribute must be used with either an IDurableClient, IDurableEntityClient, or an IDurableOrchestrationClient</value>
+  </data>
+  <data name="DurableClientAnalyzerMessageFormat" xml:space="preserve">
+    <value>DurableClient attribute is applied to a '{0}' but must be used with either an IDurableClient, IDurableEntityClient, or an IDurableOrchestrationClient</value>
+  </data>
+  <data name="DurableClientAnalyzerTitle" xml:space="preserve">
+    <value>DurableClient attribute must be used with either an IDurableClient, IDurableEntityClient, or an IDurableOrchestrationClient</value>
+  </data>
+  <data name="EntityClassNameAnalyzerDescription" xml:space="preserve">
+    <value>Entity class name and Entity function name must match</value>
+  </data>
+  <data name="EntityClassNameAnalyzerMessageFormat" xml:space="preserve">
+    <value>Class name ('{0}') and entity function name ('{1}') do not match</value>
```

Comment:
In other places you just use single quotes `'{0}'`, but here you're using both parenthesis and single quotes `('{0}')`. I suggest we just use the former for all of our messages.